zenoh-ts/src/remote_api/session.ts

```diff
@@ -135,6 +139,19 @@ export class RemoteSession {
   //
   // Zenoh Session Functions
   //
+  // Info
+  async info(): Promise<SessionInfoIface> {
+    let ctrl_message: ControlMsg = "SessionInfo";
+    this.session_info = null;
+    this.send_ctrl_message(ctrl_message);
+
+    while (this.session_info === null) {
```

Comment:
Does this code means that if session info is not arrive immediately, it will wait for 10 seconds until next try?
I believe there is a way to notify asynchronous function exactly at the moment when session_info is updated - by creating and awaiting for promise which callback is called at the moment when `session_info` is set.
Something like this - is it possible?
https://stackoverflow.com/questions/53900575/how-to-create-an-async-function-that-waits-on-an-event-in-javascript